cloudbuild.yaml

```diff
@@ -1,3 +1,12 @@
+#############################################################################
```

Comment:
The other 2 files ("cloudbuild_master.yaml" and "cloudbuild_tag.yaml") have almost zero comments, and don't explain why they are needed. Is there a doc somewhere?

Reply:
They manage/test kube configs for the CI/CD env, I don't know if there are docs but we should make some if not.